src/gestures/gesture-controller.ts

```diff
@@ -191,10 +191,7 @@ export class GestureController {
 
   isDisabled(gestureName: string): boolean {
     let disabled = this.disabledGestures[gestureName];
-    if (disabled && disabled.size > 0) {
-      return true;
-    }
-    return false;
+    return !!(disabled && disabled.size > 0);
```

Comment:
Maybe.. while this is correct for other languages, JS will do weird things.
`0 && true` and `1 && 'true'` will both not return a boolean for example. The `if` was there to make sure it will always return a boolean, so I added a cast.